e2eAIOK/DeNas/nlp/utils.py

```diff
@@ -175,7 +175,7 @@ def bert_create_optimizer(model, cfg):
         num_train_optimization_steps = int(
             cfg.num_train_steps / cfg.gradient_accumulation_steps) * cfg.train_epochs
         if ext_dist.my_size > 1:
-            num_train_optimization_steps = num_train_optimization_steps // ext_dist.my_size
+            num_train_optimization_steps = num_train_optimization_steps * ext_dist.my_size

```

Comment:
@tianyil1 , using // or * seams to be a big change, can you explain here?

Reply:
Yes, in the distributed performance test session ("ext_dist.my_size" will not be "1"), the previous setting will cause no training update on a few last training steps, which causes the performance regression on the distributed training. Thus, I fix this setting, which is tested well in my current distributed test.

Reply:
Got it, thx